index.html

```diff
@@ -13,19 +13,39 @@
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
         <link rel="stylesheet" href="style.css">
     </head>
+
     <body>
+        <!--navbar fixed top-->
+        <nav class="navbar navbar-expand-lg navbar-light bg-light fixed-top">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#">Awesome Books</a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+                <div class="collapse navbar-collapse mx-auto" id="navbarNavAltMarkup">
+                    <div class="navbar-nav ml-auto">
+                        <a onclick="Show('list')" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" id="listLink" class="nav-link active" aria-current="page" href="#">List</a>
+                        <a onclick="Show('add')" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" id="addLink" class="nav-link" href="#">Add new</a>
+                        <a onclick="Show('contact')" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" id="contactLink" class="nav-link" href="#">Contact</a>
+                    </div>
+                </div>
+            </div>
+        </nav>
 
-        <h2>All awesome books</h2>
-        <!--content generated with js TABLE-->
-        <table class="table border" id="booksTable">
-           
-        </table>
+        <!--Datetime-->
```

Comment:
- [ ] OPTIONAL
There are comments found in your code on the repository. Kindly endeavor to remove **all** comments from your code before pushing it to the repository so as to keep your code cleaner.

Reply:
changed this comments to regions, to keep the code simple to read